src/ServicePulse.Host.Tests/VerifyAppConstantsJSTextReplacement.cs

```diff
@@ -19,9 +21,9 @@ public class VerifyAppConstantsJSTextReplacement
         [Test]
         public void app_constants_js_validation()
         {
-            var pathToConfig = Path.Combine("app.constants.js");
+            var pathToConfig = Path.Combine(TestContext.CurrentContext.TestDirectory, "app.constants.js");
```

Comment:
I checked with vs nunit adapter and it uses as a work directory:
C:\PROGRAM FILES (X86)\MICROSOFT VISUAL STUDIO 14.0\COMMON7\IDE\COMMONEXTENSIONS\MICROSOFT\TESTWINDOW
So the test failed. My change make it work for this 2 test runners.